test/util/util.go

```diff
@@ -122,7 +123,7 @@ func SetupKubestellar(releasedFlag bool) {
 	if releasedFlag {
 		args = []string{"--released"}
 	}
-	commandName := "../common/setup-kubestellar.sh"
+	commandName := "../../e2e/common/setup-kubestellar.sh"
```

Comment:
What about the other call on Cleanup() , with a different working directory? (will not be an issue if the new test is not put in a "cicd" subdirectory.)

Reply:
My concern has been alleviated by the change in directory structure. Originally, Cleanup was called from two different levels of directory nesting, so that there was no correct constant relative directory navigation to use here.